source/_dashboards/statistics-graph.markdown

```diff
@@ -30,7 +30,7 @@ type:
   type: string
 entities:
   required: true
-  description: "A list of entity IDs or `entity` objects (see below), or an external statistic id"
+  description: "A list of entity IDs or `entity` objects (see below), or an external statistic id. Note: After specifying the first one, all others can only be selected with the same unit of measurement."
```

Comment:
This describes the YAML configuration parameters, there is no selection or UI in such cases? I'm confused about what that means in YAML?

Reply:
Hi, I don't use YAML, trying to do stuff in the UI. So I don't know what is happening if you select entities with different unit of measurements via YAML. I can only speak for the UI. There the entities are filtered right after you select the first entities. After that, only entities of the same unit of measurements are shown. The card doesn't behave like the history graph card. There is no such restriction and a new graph is added with the new unit.

I tried in yaml, and the behavior is more than strange: 
I can add another entity with another uni there, but it is shown in the same graph with the wrong unit (= unit of the first entity). 

![image](https://user-images.githubusercontent.com/66269310/208655210-df690ca1-d2e9-4d99-926c-889d2f66b833.png)

Power entity only: 
![image](https://user-images.githubusercontent.com/66269310/208655339-f0debeec-d9fa-447a-b865-5594f21cd6e5.png)

History graph card: 
![image](https://user-images.githubusercontent.com/66269310/208655570-e502f3fe-bc2f-425b-9152-0b153fc98f66.png)

cannot select power entity via UI:
![image](https://user-images.githubusercontent.com/66269310/208655801-b1bd869c-200d-4b90-807d-c80825d94015.png)

can select power entity via UI with history graph card:
![image](https://user-images.githubusercontent.com/66269310/208656058-59866d14-c9ed-4cc6-a428-f13759a329f7.png)

What do you think?

Reply:
> Hi, I don't use YAML, trying to do stuff in the UI.

You suggested change is made on the YAML configuration default in the documentation, hence my comment on this PR. 
The rest of your UI-based comments are thus not relevant 🤷

Reply:
Then we leave it as it is. 
There is no section for UI in the documentation right now. 

> The rest of your UI-based comments are thus not relevant 🤷
I wrote that, because the behavior via yaml is inconsistent as well, but in a different way.
Should I open an issue for that or does it work as designed from your point of view? (still learning)

Reply:
btw. in the documentation it doesn't say YAML, just `CONFIGURATION VARIABLES` after describing how to add the card via UI. Later, below `Alternatively, the card can be configured using YAML`. 

So I read it the way, that the standard is a configuration via "UI" and alternatively you can do it via YAML. So the variables are at least for both ways (maybe with additional infos like `list` ) for people doing it via YAML.